modules/storages/app/common/storages/peripherals/oauth_configurations/one_drive_configuration.rb

```diff
@@ -52,6 +51,20 @@ def authorization_state_check(access_token)
           end
         end
 
+        def extract_origin_user_id(rack_access_token)
+          Net::HTTP.start(@uri.host, @uri.port, use_ssl: true) do |http|
+            response = http.get('/v1.0/me',
+                                { Authorization: "Bearer #{rack_access_token.access_token}", Accept: 'application/json' })
+
+            # if response == Net::HTTPSuccess
```

Comment:
@mereghost Are those comments supposed to stay here?

Reply:
Yes and no. Not as comments, but as code. :P